kit.yml

```diff
@@ -191,7 +189,9 @@ credentials:
   blacksmith-integration:
     users/blacksmith:
       password: random 30
-
+  prometheus-integration:
+    users/prometheus:
```

Comment:
Is there a reason this wasn't kept as `uaa/clients/prometheus:secret`?  It may break any existing deployments if you change this.